### ISSUE TITLE
[R: compile error] The system cannot find the path specified.

### ISSUE THREAD

**Issue by benyamindsmith** [FabrizioSandri/r-integration#4]
First off, I'm really excited so see that such a package has been made, I am really looking forward to trying this out!

My problem is the following. 

I have the following script:

```js
const R = require("r-integration");
let result = R.executeRCommand("print('hello')");
console.log(result);
```
but after trying to run this code I get the following output. 

`[R: compile error] The system cannot find the path specified.`

I read through the documentation and have done some googling and haven't really found a solution. Probably its something obvious that I have overlooked. Where the issue could be and where the R path should be defined? 

Best. 

Benjamin

**Comment by FabrizioSandri** [comment]
Good catch! I think it was a problem related to the default path where R was installed. Remember to install R before using this library.
I've just released the new version 1.5.0 which allow you to specify al alternative location for the binaries with the `RBinariesLocation` parameter. Let me know if this version solves your error.

For more, check the new [documentation paragraph](https://github.com/FabrizioSandri/r-integration#alternative-r-binaries-location)

**Comment by benyamindsmith** [comment]
Just reinstalled the package and now I have this error for running this code: 

```js
const R = require("r-integration");

let result = R.executeRCommand("max(1,2,3)", "C:\\Program Files\\R");
console.log(result);
```

`

PS D:\RandJavascript> node RScriptInJavaScript

internal/fs/utils.js:620
    throw new ERR_INVALID_ARG_TYPE(propName, ['string', 'Buffer', 'URL'], path);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received null
    at Object.readdirSync (fs.js:1016:10)
    at isRscriptInstallaed (D:\RandJavascript\node_modules\r-integration\src\R.js:69:37)
    at Object.executeRCommand (D:\RandJavascript\node_modules\r-integration\src\R.js:110:29)
    at Object.<anonymous> (D:\RandJavascript\RScriptInJavaScript.js:3:16)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'ERR_INVALID_ARG_TYPE'
}
`

**Comment by benyamindsmith** [comment]
Update: I changed my path and now I have this:
```js
const R = require("r-integration");

let result = R.executeRCommand("max(1,2,3)", "C:\\Program Files\\R\\R-4.1.0\\x64");
console.log(result);
```

Terminal output:

```
PS D:\RandJavascript> node RScriptInJavascript
R not found, maybe not installed.
See www.r-project.org
```

Which is strange as the binary files are there.

**Comment by FabrizioSandri** [comment]
In your case the installation directory should be `C:\\Program Files\\R`

**Comment by FabrizioSandri** [comment]
@benyamindsmith In your case the installation directory should be `C:\\Program Files\\R`. The first one was correct, not the one followed by `R-4.1.0\\x64`

I just resolved the problem in the library and released a new version 1.6.0. 
Update to this new version. I'm sorry for the problem.

**Comment by benyamindsmith** [comment]
@FabrizioSandri  just got it to work for me now!

Definitely gonna share this around with other R users. 

Thank you for making this!
